### DISCUSSION TITLE
Hard time compiling ESPAsync_WIFIManager using Platformio

### DISCUSSION THREAD

**Discussion by nicolas-rabault** [khoih-prog/ESPAsync_WiFiManager#109] [Q&A]
Hey folks,

I'm trying to use ESPAsync_WIFIManager to create a wifi example on the [Luos_engine](https://github.com/Luos-io/luos_engine) lib. But I have hard time to solve all the dependancies issue.

To make it simple I came back to a simple project. I'm just tying to compile the `Async_ConfigOnDoubleReset` example for ESP32.
I use the `platformio.ini`provided on your repo without changing anything :
![Screenshot 2022-08-23 at 12 16 36](https://user-images.githubusercontent.com/9554146/186133697-36c3a0bd-5397-4cef-a7dd-3154b5fa4af5.png)
Here is the error I get : 
```
src/Async_ConfigOnDoubleReset.cpp:359:10: fatal error: ESPAsync_WiFiManager.h: No such file or directory

******************************************************************************
* Looking for ESPAsync_WiFiManager.h dependency? Check our library registry!
*
* CLI  > platformio lib search "header:ESPAsync_WiFiManager.h"
* Web  > https://registry.platformio.org/search?q=header:ESPAsync_WiFiManager.h
*
******************************************************************************

 #include <ESPAsync_WiFiManager.h>               //https://github.com/khoih-prog/ESPAsync_WiFiManager
```

So I added `https://github.com/khoih-prog/ESPAsync_WiFiManager` as a dependancy on the platformio.ini, and now i have : 
```
.pio/libdeps/ESP32/ESPAsync_WiFiManager/src/ESPAsync_WiFiManager.hpp:208:10: fatal error: ESPAsyncDNSServer.h: No such file or directory

***************************************************************************
* Looking for ESPAsyncDNSServer.h dependency? Check our library registry!
*
* CLI  > platformio lib search "header:ESPAsyncDNSServer.h"
* Web  > https://registry.platformio.org/search?q=header:ESPAsyncDNSServer.h
*
***************************************************************************

 #include <ESPAsyncDNSServer.h>
```

So again I added `devyte/ESPAsyncDNSServer@^1.0.0` as a dependancy but I still have the same error...

Did I miss something?

**Comment by khoih-prog** [comment]
Check [**ESPAsyncDNSServer depends on faulty and not maintained library ESPAsyncUDP** #108](https://github.com/khoih-prog/ESPAsync_WiFiManager/issues/108)
